asv_bench/benchmarks/indexing.py

```diff
@@ -358,6 +358,10 @@ def time_assign_with_setitem(self):
         for i in range(100):
             self.df[i] = np.random.randn(self.N)
 
+    def time_assign_list_like_with_setitem(self):
```

Comment:
do we also have the concat example (you can put it here)?

Reply:
Done